src/Microsoft.DotNet.Darc/DarcLib/BarApiClient.cs

```diff
@@ -20,11 +20,11 @@ public class BarApiClient : IBarApiClient
 {
     private readonly IMaestroApi _barClient;
 
-    public BarApiClient(string? buildAssetRegistryPat, string? federatedToken, bool disableInteractiveAuth, string? buildAssetRegistryBaseUri = null)
+    public BarApiClient(string? buildAssetRegistryPat, string? federatedToken, bool disableInteractiveAuth, string cachePath, string? buildAssetRegistryBaseUri = null)
```

Comment:
Putting it here like this is a nasty breaking change for everyone setting the `buildAssetRegistryBaseUri` parameter

Reply:
Removed